testit-adapter-playwright/src/converter.ts

```diff
@@ -57,29 +56,35 @@ export class Converter {
         autotestResult.traces = status.trace;
       }
 
+      if (autotestData.addMessage) {
+        autotestResult.message = autotestData.addMessage
+      }
+
       return autotestResult;
     }
 
     static convertTestStepsToShortSteps(steps: TestStep[]): ShortStep[] {
-      return steps.map(step => {
-        return this.convertTestStepToShortStep(step);
-      });
+      return steps.filter((step: TestStep) => step.category === "test.step").map(step => this.convertTestStepToShortStep(step));
```

Comment:
Тяжело читать, как насчет: 

```suggestion
      return steps
        .filter((step: TestStep) => step.category === "test.step")
        .map(step => this.convertTestStepToShortStep(step));
```